### PR TITLE
Library upgrade

### PR DESCRIPTION
**Pull request by adkumar1**
- Spring boot library update

### CODE REVIEW

**Comment by Wulghash** [comment]
@Siegfriedk Please check this PR swell

File: DEPENDENCIES [hunk 1]
```diff
@@ -3,16 +3,16 @@ maven/mavencentral/ch.qos.logback/logback-core/1.4.7, EPL-1.0 OR LGPL-2.1-only,
...
-maven/mavencentral/com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.14.2, Apache-2.0, approved, #8162
-maven/mavencentral/com.fasterxml.jackson.module/jackson-module-parameter-names/2.14.2, Apache-2.0, approved, #5938
+maven/mavencentral/com.fasterxml.jackson.core/jackson-annotations/2.15.0, Apache-2.0, approved, #7947
+maven/mavencentral/com.fasterxml.jackson.core/jackson-core/2.15.0, MIT AND Apache-2.0, approved, #7932
+maven/mavencentral/com.fasterxml.jackson.core/jackson-databind/2.15.0, Apache-2.0, approved, #7934
+maven/mavencentral/com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.15.0, Apache-2.0, approved, #8802
+maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.15.0, Apache-2.0, approved, #8808
+maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.15.0, Apache-2.0, approved, #7930
+maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.15.0, Apache-2.0, approved, clearlydefined
+maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.15.0, Apache-2.0, restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES:13]: Please open IP issue tickets in Eclipse and update all the restricted libraries with ticket ID. If you don't have a contributor with Eclipse GitLab personal access token then let us know and we'll open the tickets.

Comment (almadigabor) [reply DEPENDENCIES]: Opened the tickets and updated the DEPENDENCIES file with the approved lines.

**Comment by adkumar1** [comment]
Hi Sigi,
I have updated the version

**Review by almadigabor — APPROVED** [review]
LGTM

**Pull request merged**